### ISSUE TITLE
PySide6: export dialog does not show param values

### ISSUE THREAD

**Issue by pijyoi** [pyqtgraph/pyqtgraph#1728]
### Short description

With PySide6, the export dialog does not show the params' values. The other 3 bindings work fine.

### Code to reproduce

launch examples/SimplePlot.py and activate the export dialog.

### Expected behavior

![good](https://user-images.githubusercontent.com/2657027/115426314-f658a080-a232-11eb-81ea-fd0041da0d30.png)

### Real behavior

![bad](https://user-images.githubusercontent.com/2657027/115426339-fce71800-a232-11eb-920e-bbdbc09873da.png)

### Tested environment(s)

 * PyQtGraph version: 0.12.1.
 * Qt Python binding: PySide6 6.0.2 Qt 6.0.2
 * Python version: 3.8.9
 * NumPy version: 1.20.2 
 * Operating system: Windows 10 (Also Linux with similar major.minor versions)
 * Installation method: pip install -e . 

### Additional context

As far as I can tell, this issue has been present since the addition of PySide6 support (i.e. 78bc0fd3ca206a60e0ee2a1095379bee869485e3).
examples/parametertree.py seems to work fine.

**Comment by j9ac9k** [comment]
now that ... is odd.  Wonder if that's a pyside6 bug.

**Comment by pijyoi** [comment]
If I manually edit exportDialogTemplate_pyside6.py from
```python
        self.paramTree = ParameterTree(Form)
        __qtreewidgetitem1 = QTreeWidgetItem()
        __qtreewidgetitem1.setText(0, u"1");
        self.paramTree.setHeaderItem(__qtreewidgetitem1)
        self.paramTree.setObjectName(u"paramTree")
        self.paramTree.header().setVisible(False)
```
to
```python
        self.paramTree = ParameterTree(Form)
        self.paramTree.setObjectName(u"paramTree")
        self.paramTree.headerItem().setText(0, u"1")
        self.paramTree.header().setVisible(False)
```
then the parameter values show up.

Conversely, if I were to modify the PySide2 generated file to look like the PySide6 version, then the parameter values won't show up. So the issue is that pyside6-uic.exe (compared to the other 3 bindings) generates code that trigger some bug / mis-feature in Qt.

Not sure how to proceed from here.

**Comment by j9ac9k** [comment]
Probably should commit the manual changes to the template; but of course that's not a long-term fix.

Does pyside6 offer a loadUi functionality?  If they do we maybe should consider trying to use that to load the `.ui` file directly instead of loading the template.

**Comment by pijyoi** [comment]
I found that the issue is also present with pyside2-uic.exe. The current in-tree generated pyside2 template file was generated with a much older version
```
# Created: Sun Sep 18 19:19:58 2016
#      by: pyside2-uic  running on PySide2 2.0.0~alpha0 
```
If I regenerate it with pyside2-uic.exe from PySide2 5.15, then I get the same generated code as pyside6-uic.exe.

How about taking one of the PyQt generated template files and modifying it such that a single file is usable by all bindings? It should only involve changing the binding imports. The last change to the .ui file was in 2013/12 a2e8290d8e223a630e77708e7f5c3ba42ddd13ee

**Comment by j9ac9k** [comment]
I was curious if the loadUi capability would work, but it does not:

```python
from PySide6.QtUiTools import QUiLoader
from PySide6.QtCore import QFile
import pyqtgraph as pg

pg.mkQApp()

ui_file = QFile("pyqtgraph/GraphicsScene/exportDialogTemplate.ui")
ui_file.open(QFile.ReadOnly)
loader = QUiLoader()
dialog = loader.load(ui_file)
dialog.show()

if __name__ == "__main__":
    pg.mkQApp().exec_()
``` 

during the run, it generated the following warning:

```
"QFormBuilder was unable to create a custom widget of the class 'ParameterTree'; defaulting to base class 'QTreeWidget'."
```

>  How about taking one of the PyQt generated template files and modifying it such that a single file is usable by all bindings? It should only involve changing the binding imports. 

I think this is likely what has to be done; I don't know of any other solution here.  I suspect this will likely need to happen for all the template files (but no need to do them all in 1 PR, we can wait as issues arise).

**Comment by j9ac9k** [comment]
Fixed with #1739
